templates/stamp/run-customizations.sh

```diff
@@ -411,8 +411,22 @@ if [ "$MACHINE_ROLE" == "jumpbox" ];
 then
     log "Starting backup configuration on '${HOSTNAME}' as a member in the '${MACHINE_ROLE}' role"
 
-    # setup the configuration file for database backups
-    source "${OXA_ENV_PATH}/${DEPLOYMENT_ENV}.sh"
+    # setup the configuration file for database backups.
+    config_file="${OXA_ENV_PATH}/${DEPLOYMENT_ENV}.sh"
+
+    # get BASE_URL value
+    source $config_file
+    exit_on_error "Failed sourcing the environment configuration file from keyvault" 1 "${MAIL_SUBJECT} Failed" $CLUSTER_ADMIN_EMAIL $PRIMARY_LOG $SECONDARY_LOG
+
+    # replace "deployment-time" values. Persist to file.
+    export BASE_URL=$BASE_URL
+    export AZURE_ACCOUNT_NAME=$BACKUP_STORAGEACCOUNT_NAME
+    export AZURE_ACCOUNT_KEY=$AZURE_ACCOUNT_KEY
+    # todo: use cluser name to drive replication values, etc.
```

Comment:
- [ ] add the other values that need to be replaced and move the replacement to a function in utilities.sh

1. Azure Active Directory OAuth2 Third Party Authentication Configuration
2. Comprehensive Theming Configuration
3. Import Kitchen Sink Course Configuration